### DISCUSSION TITLE
Using InputCount component

### DISCUSSION THREAD

**Discussion by VershininNV** [taiga-family/taiga-ui#1170] [Q&A] [answered]
I am newbie in front development, so I use this component and would like to change the size of the buttons in this component. If I understood correctly, InputCount consists of tui-primitive-textfield and section in which the buttons are located, how could I change their height and width to make it look something like this...
![component](https://user-images.githubusercontent.com/93840904/148174669-9e257b9e-7fa6-45a5-a82d-0801860e2328.png)

**Comment by waterplea** [comment, marked as answer]
This part is not customizable without hacks. You can use ::ng-deep or some global styles to hack width of a `button` inside `tui-input-count`. Be sure your styles have more specificity than built-in, probably will need `!important`.

**Reply by VershininNV** [reply]
Thank you for fast reply, your advice to use ::ng-deep helped me.
